app/javascript/courses/review/components/CoursesReview__ShowReviewedSubmissions.re

```diff
@@ -64,13 +64,20 @@ let updateReviewedSubmissions =
   setState(_ => Loaded);
 };
 
+let sortDirectionAsString = sortDirection =>
+  switch (sortDirection) {
+  | `Up => "ASC"
+  | `Down => "DESC"
+  };
```

Comment:
This conversion will be unnecessary if `sortDirection` is an _enum_ according to GraphQL.